src/details/ArborX_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -212,7 +212,7 @@ struct DistributedSearchTreeImpl
   template <typename OutputView>
   static void communicateResultsBack(
       MPI_Comm comm, OutputView &view, Kokkos::View<int *, DeviceType> offset,
-      Kokkos::View<int *, DeviceType> &ranks,
```

Comment:
In view of #217, I guess you want to drop 9ce7ba7a27a829edea00034face71c3e7844cdc4 here.

Reply:
At some point. However, I think it  may be impossible if the callbacks are implemented only for spatial and not for nearest.